### ISSUE TITLE
Syntax highlighting not working

### ISSUE THREAD

**Issue by rpahl** [privefl/jekyll-now-r-template#1]
Hi,
I have set up everything according to your readme and everything works just fine except the syntax highlighting, which does not seem to work properly as everything gets rendered blue, for example:

![code-snippet](https://user-images.githubusercontent.com/11405819/67164099-f7c3b700-f376-11e9-8b75-c01f2d58b13c.png)

Here is your minimal example on my github page: [a-minimal-r-markdown-example](https://rpahl.github.io/blog/a-minimal-r-markdown-example/) and [here](https://rpahl.github.io/blog/container-deque-set-and-dict-for-r/) another post.

I guess, this is most likely a css-style (which I did not touch) issue but maybe you have an idea?

Many thanks in advance,
Roman

**Comment by privefl** [comment]
I tried to reproduce your problem:
https://privefl-demo.github.io/blog/a-minimal-r-markdown-example/
But seems OK.

Could I ask you to try again from scratch?

**Comment by privefl** [comment]
You have a new file `style.css` that I don't have. Is it normal?

**Comment by rpahl** [comment]
First of, thanks for your very quick reply! :)

> You have a new file `style.css` that I don't have. Is it normal?

I think, I copied a style.css from one of your blog posts to try something else.

> Could I ask you to try again from scratch?

Re-did it from scratch now - still the same issue ;/

**Comment by privefl** [comment]
Just try only the following minimal steps: 
 - renaming to "blog" and enabling GitHub pages
 - pulling as RStudio project
 - `prettyjekyll::FormatPost("_knitr/knitr-minimal.Rmd")`
 - commit + push

(Don't modify `_config.yml` yet.)

**Comment by rpahl** [comment]
Done (issue remains).

**Comment by privefl** [comment]
Ok, what do you have for `sapply(c("prettydoc", "prettyjekyll"), packageVersion)`?

**Comment by rpahl** [comment]
```{r}
sapply(c("prettydoc", "prettyjekyll"), packageVersion)
$prettydoc
[1] 0 3 0

$prettyjekyll
[1] 0 2 3
```

**Comment by privefl** [comment]
Same as me..

And R and RStudio?

I'm running out of ideas :/

**Comment by rpahl** [comment]
It's R 3.6.1 and RStudio 1.2.1564 (committed both via RStudio and console) but, if anything, I assume, it should be probably more about the knitr version, right?

Anyway, never mind. I guess I'll have to dig a little deeper into the css and/or html stuff then :)

Again, thanks for your quick replies! At least, I am more confident now, that I did'nt something stupid ;)

**Comment by privefl** [comment]
Sorry for not being useful here.
I don't have time for now to look deeper into this.
If you find the problem/solution, please report it here for others.

**Comment by rpahl** [comment]
> If you find the problem/solution, please report it here for others.

Probably also not have time before next weekend, but if I solve it, I'll sure post it here.

**Comment by rpahl** [comment]
I had a look and I think, I have narrowed it down. Basically, the produced markdown code of your and mine version differ.
Your version:
```html
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">curve</span>(<span class="kw">exp</span>(<span class="op">-</span>x<span class="op">^</span><span class="dv">2</span>), <span class="op">-</span><span class="fl">2.5</span>, <span class="fl">2.5</span>)</code></pre></div>
```

My version:
```html
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1"></a><span class="kw">curve</span>(<span class="kw">exp</span>(<span class="op">-</span>x<span class="op">^</span><span class="dv">2</span>), <span class="fl">-2.5</span>, <span class="fl">2.5</span>)</span></code></pre></div>
```

So I guess, you are using a different pandoc compiler than me. What's your OS?

**Comment by privefl** [comment]
I'm on Linux CentOS 7 most of the time.

**Comment by privefl** [comment]
```{r}
> rmarkdown::pandoc_version()
[1] ‘1.19.2.1’
```

**Comment by rpahl** [comment]
I checked - indeed it's the difference in pandoc versions. I have used the latest release (2.7), which caused the difference. When using the older version (e.g. 1.19.2.1) the syntax highlighting works, so I guess, one either has to use this pandoc version or adapt the css file.

**Comment by privefl** [comment]
I think I remember something of having Pandoc >= 2 when you get RStudio >= 1.2.
I'm still using 1.1.x.

So, the problem would be that pandoc is not compatible with older versions?
When I have some time, I will look at {prettydoc}, which I'm using under the hood.

**Comment by rpahl** [comment]
> I think I remember something of having Pandoc >= 2 when you get RStudio >= 1.2.

Yes, RStudio is shipped with its own (mostly somewhat recent) version of pandoc, currently 2.7.2.
